content/content-tool/tool/src/webapp/vm/content/sakai_resources_list.vm

```diff
@@ -80,7 +80,7 @@
 		</ul>
 		</ul>
 	#if ($alertMessage)
-                 <br /><div class="messageValidation">$alertMessage</div><div class="clear"></div>
+                 <br /><div class="alertMessage">$alertMessage</div><div class="clear"></div>
```

Comment:
This seems like the right change to make, did you notice any other code using this block?

Reply:
All the other velocity pages that use the alertMessage in the context have the class as alertMessage.